x-pack/plugins/apm/public/components/shared/Links/MachineLearningLinks/MLManageJobsLink.tsx

```diff
@@ -34,7 +34,8 @@ export function MLManageJobsLink({ children, external }: Props) {
     rangeFrom = 'now-1h',
     rangeTo = 'now',
     refreshInterval = timePickerRefreshIntervalDefaults.value,
-    refreshPaused = timePickerRefreshIntervalDefaults.pause,
+    // defaults it to true to avoid infinite loop when refreshInterval is "0"
+    refreshPaused = true,
```

Comment:
I think the ML team is fixing this separately: https://github.com/elastic/kibana/pull/86805. I'm not sure if this is necessary then.

Reply:
great will revert then